components/Observations.tsx

```diff
@@ -51,6 +50,7 @@ export const Observations: React.FunctionComponent<{
     );
   }
   if (!observations.getObservationList || observations.getObservationList.length === 0) {
+    // TODO: when cleaning this up, fix it so that it renders the date in the user's locale, not UTC date
```

Comment:
I ran this at around 5 PM on 1/18 and the error message referred to the next day, since in UTC it was already the next day. This might be confusing to a user: 

<img width="356" alt="image" src="https://user-images.githubusercontent.com/101196/213339419-c5991b22-b80b-4f7c-810c-eb896301b734.png">